code/_onclick/hud/rendering/plane_master_group.dm

```diff
@@ -34,6 +34,14 @@
 		stack_trace("Hey brother, our key [key] is already in use by a plane master group on the passed in hud, belonging to [viewing_hud.mymob]. Ya fucked up, why are there dupes")
 		return
 
+#if MIN_COMPILER_VERSION > 515
+	#warn Fully change default relay_loc to "1,1", rather than changing it based on client version
+#endif
```

Comment:
this wouldn't actually be valid, since we could have clients from older versions connecting. If you want to warn for this, do it off > 516 since that guarentees a reasonable level of safety.

that or just leave a big comment and we can remember to deal with it someday